test/geometry_tests.cpp

```diff
@@ -296,6 +298,89 @@ TEST(raytrace, negative_length) {
 	EXPECT_EQ(pixels.size(), 0);	
 }
 
+struct PolygonScenario{
```

Comment:
Can we get doxygen comments?

Reply:
Added, I think?

Reply:
They're not params.
Do like
```
/**
 * @brief Creates struct for testing polygonOutline
 */	
struct PolygonScenario{
  std::vector<Cell> vertices;           /**< The vertices of the polygon to create */
  int size_x;                           /**< Maximum length of a polygon side */
  int expected_size;                    /**< Expected number of cells for the outline */
  std::vector<Cell> expected_elements;  /**< Expected cells for the outline */
};
```

See the Javadoc style https://www.doxygen.nl/manual/docblocks.html